### DISCUSSION TITLE
1.6.20

### DISCUSSION THREAD

**Discussion by camueller** [camueller/SmartApplianceEnabler#225] [Releases]
Details zu den Änderungen finden sich im [Changelog](https://github.com/camueller/SmartApplianceEnabler/blob/master/CHANGELOG.md#1620---15032022),

**Comment by Trunks1982** [comment]
Was mir aufgefallen ist bei dem GO E wenn ich 230v angebe komme ich bei genügend Überschuss nicht auf 32A Ladung.Testweise hab ich dann auf 225V eingestellt und dann hat er auf die maximalen 32A hochgeregelt.Tatsächlich liegen aber immer mindestens 232 V an.Kann man das nicht irgendwie immer automatisch vom SHM abgreifen?

**Comment by camueller** [comment]
Der SHM gibt die Leistung in Watt vor. Der SAE rechnet diese über die Phasenanzahl in die bei der Wallbox zu setzende Stromstärke um. Die Berechnung findet sich im Log. Dort würde ich zuerst prüfen, ob die errechnete Stromstärke Deinen Erwartungen entspricht. Danach kannst Du prüfen, ob diese auf der Wallbox korrekt gesetzt wurde.

**Comment by Trunks1982** [comment]
Ok kann ich mal machen.Aber lass jetzt auf 225V erstmal laufen da passt es. Aso 1 Phase hab ich eingestellt.
